conan/tools/cmake/toolchain/blocks.py

```diff
@@ -265,7 +265,7 @@ def context(self):
         linker_scripts = [linker_script.replace('\\', '/') for linker_script in linker_scripts]
         linker_scripts = [relativize_path(p, self._conanfile, "${CMAKE_CURRENT_LIST_DIR}")
                           for p in linker_scripts]
-        linker_script_flags = ['-T"' + linker_script + '"' for linker_script in linker_scripts]
+        linker_script_flags = ['-T' + linker_script for linker_script in linker_scripts]
```

Comment:
Are you sure these quotes are to be removed? They can be very necessary when there are path containing spaces, can you please try to keep them?

Reply:
You already found the fix apparently :D